Donation-platform/src/main/webapp/personalizationScript.js

```diff
@@ -107,18 +109,28 @@ function placeSelectedTagInRank(tag) {
     const tagRank = document.getElementById('sortable');
 
     let tagElements = new Map([
```

Comment:
If possible can we construct these elements with javascript. Using innerHTML in this way can be hard to read and is also prone to errors. 
https://stackoverflow.com/questions/11515383/why-is-element-innerhtml-bad-code

Reply:
Agreed. Please look into it as you will have a template to quickly make more and avoid copying errors.

Reply:
Done.